src/Components/Politicians/PoliticianCard/index.js

```diff
@@ -4,6 +4,7 @@ import Paper from 'material-ui/Paper';
 import { Link } from 'react-router-dom';
 import LinearProgress from 'material-ui/LinearProgress';
 import type { PoliticianType } from '../../../types';
+import * as styles from './styles';
```

Comment:
gah saw this commit now :S

If you're interested, I can show you of the ways I rewrite git history.

To avoid exactly these kind of situations